tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -0,0 +1,27 @@
+package e2e
+
+import (
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/cloudprovider"
+	"testing"
+	"time"
+)
+
+func TestE2E(t *testing.T) {
+	aws, err := cloudprovider.CloudProviderFactory("/tmp/kubeconfig", "/tmp/awscredentials", "default",
+		"/tmp/aws/")
+	if err != nil {
+		t.Errorf("Error creating clients, %v", err)
+	}
+	err = aws.CreateWindowsVM("ami-04ca2d0801450d495", "m4.large", "libra")
+	if err != nil {
```

Comment:
Please use asserts in testing. Moreover not getting err is not sufficient. You need to query the provider to check if the resources with appropriate configuration was created.